app/main.py

```diff
@@ -1,13 +1,26 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        "line": error.get("line_number", None),
```

Comment:
If you are sure that the` line_number` key will always be present in the error dictionary, you can use 
`"line": error["line_number"]` instead of `"line": error.get("line_number", None)`. But it’s up to you.

Reply:
no need to write None as default arg, it will return None by default
fix this everywhere

Reply:
no need to write None as default arg, it will return None by default
fix this everywhere